### PR TITLE
Add after each function to avoid TC to fail due to use the wrong user

### PR DESCRIPTION
**Pull request by aboughdiri**

### CODE REVIEW

**Comment by aboughdiri** [comment]
shoudn't it be done in Base class to avoid duplicate the code in every test class ?

Your are right, done

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java [hunk 1]
```diff
@@ -40,6 +42,14 @@ public static void setup() {
 
   }
 
+  @AfterEach
+  public void signout() {
```
Comment (thomasdelhomenie) [inline commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: I would move this method under openPlatform method to make it more readable (before/after).

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java [hunk 2]
```diff
@@ -40,6 +42,14 @@ public static void setup() {
...
+    ManageLogInOut manageLogInOut;
+    manageLogInOut=new ManageLogInOut(this);
```
Comment (thomasdelhomenie) [inline commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: One line is enough : ManageLogInOut manageLogInOut = new ManageLogInOut(this);

Comment (aboughdiri) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: done

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java (hunk 1 above)
Comment (aboughdiri) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: done

**Pull request merged**